src/model/commands/assets.rs

```diff
@@ -1,22 +1,24 @@
-use crate::model::commands::isi::Command;
+use crate::model::commands::isi::{Command, Id};
 
 /// The purpose of add asset quantity command is to increase the quantity of an asset on account of
 /// transaction creator. Use case scenario is to increase the number of a mutable asset in the
 /// system, which can act as a claim on a commodity (e.g. money, gold, etc.).
 #[derive(Clone, Debug, PartialEq, serde::Serialize, serde::Deserialize)]
 pub struct AddAssetQuantity {
-    pub asset_id: String,
+    pub asset_id: Id,
+    pub account_id: Id,
```

Comment:
Should we check that `asset` and `account` are from the same domain on creation?

Reply:
Definitely should in `validation` module. https://soramitsu.atlassian.net/browse/RH2-15